workspaces/ui-v2/src/pages/docs/EndpointRootPage.tsx

```diff
@@ -120,6 +120,14 @@ export const EndpointRootPage: FC<
   const parameterizedPathParts = thisEndpoint.pathParameters.filter(
     (path) => path.isParameterized
   );
+  const filteredQuery =
+    thisEndpoint.query && selectors.isItemVisible(thisEndpoint.query)
+      ? thisEndpoint.query
+      : null;
+  const filteredRequests = selectors.filterRemovedItems(thisEndpoint.requests);
+  const filteredResponsesByStatusCode = selectors.filterMapOfRemovedItems(
+    thisEndpoint.responsesByStatusCode
+  );
```

Comment:
Little nit you should totally ignore: "filteredX" is one of those variables that makes total sense when you write it, but when you or someone revisits later is met with "filtered.. how?". With an app being 1 massive data pipeline, isn't everything we do filtered in some way? ^^

"existingX" might make sense here, as we clean up removed items. Or maybe something else descriptive?

Reply:
I changed this to `visibleX` - agreed that filteredX could be confusing and less descriptive